tests/test_cli.py

```diff
@@ -1,14 +1,19 @@
 from west.__main__ import main
-from west.db import get_schema_version
+from west.db import get_current_schema_version
+from west.db import is_schema_initialized
 
 
 def test_version(cli_runner):
-    assert cli_runner.invoke(main, ["--version"]).output == "0.1.0"
+    assert cli_runner.invoke(main, ["--version"]).output == "West 0.1.0\n"
 
 
 def test_current_database_state(cli_runner):
 
-    result = cli_runner.invoke(main, ["show_migrations"])
+    result = cli_runner.invoke(main, [
+        "--target-version",  "1.1",
+        "--migrations-root",  "example_migrations",
+        "migrate"])
```

Comment:
Don't we need to setup a specific state before the test ? Like, ensuring a fresh DB or something ?